docs/quickstart.md

```diff
@@ -408,7 +408,11 @@ with additional API for accessing cookies by their domain or path.
 
 ## Redirection and History
 
-By default, HTTPX will follow redirects for anything except `HEAD` requests.
+By default, HTTPX will follow redirects for all HTTP methods.
+
+!!! warning
+    For anyone comes from Requests, you may be familar that `HEAD` in Requests 
+    doesn't follow redirects by default. But that is not the case in HTTPX.
```

Comment:
I'd suggest we leave that out? We already document it in the "Requests Compatibility Guide" now, and it would be duplicate info.

```suggestion
```